### DISCUSSION TITLE
Wake after sleep not working

### DISCUSSION THREAD

**Discussion by nebulorum** [manna-harbour/miryoku#310] [Q&A] [answered]
I have been using zmk-miryoku with `ZMK_CONFIG_SLEEP` for a month, worked like a charm. I added the nanoview to the board today and did a new FW with nanoview (see [commit](https://github.com/manna-harbour/miryoku_zmk/commit/c339853c3333c8e70525fdbb1b1a23b52c91eaa)). And now it sleeps but will not wake up with any key. Did a bunch of googling and could not find anything that makes sense (on this repo, zmk, reddit). Will revert to the previous FW and will check if the physical screen are causing the issue. I'm not sure this is the right forum.

**Comment by nebulorum** [comment]
Ok tested with the non niceview firmware with the screen on and the keyboard wakes normally. So something in the nanoview version of the FW is not waking up.

**Comment by manna-harbour** [comment, marked as answer]
It looks like there was a regression recently with sleep in ZMK. See https://discord.com/channels/719497620560543766/719544534500900886/1223354718005497988 on the ZMK discord (https://zmk.dev/community/discord/invite).

There's nothing Miryoku-specific with sleep or nice view that would cause failure to wake from sleep, so if you still have issues try asking in the help channel on the ZMK discord server.

**Comment by nebulorum** [comment]
Thanks for the hint. I still don't fully understand how this build happen. I wasn't aware that the build get the latest version from ZMK on each build. Rebuilt and flashed it and it now works correctly. I need to dive deeper in this project, since I what to tweak the layout.

**Reply by manna-harbour** [reply]
For customisation see https://github.com/manna-harbour/miryoku_zmk#customisation.

By default, workflow builds checkout ZMK main for each build run. You can change the branch with https://github.com/manna-harbour/miryoku_zmk#branches. If you build locally you'll have to manage the ZMK branch manually.
